docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+shipit:
+  build: ./
+  command: python kickoff-web.py -d sqlite:///update.db --username=admin --password=password --host=0.0.0.0 --port=5000 --verbose
```

Comment:
fwiw I just learned why it was failing (with mysql locally shut off): https://github.com/mozilla/balrog/blob/master/scripts/initdb_and_run.sh#L7 -- specifically I need the insert permissions thing.